hosted_control_planes/hcp-configuring.adoc

```diff
@@ -0,0 +1,39 @@
+:_content-type: ASSEMBLY
+[id="hcp-configuring"]
+= Configuring hosted control planes
+include::_attributes/common-attributes.adoc[]
+:context: hcp-configuring
+
+
+toc::[]
+
+To get started with hosted control planes for {product-title}, you first configure your hosted cluster on the provider that you want to use. Then, you complete a few management tasks. 
+
+:FeatureName: Hosted control planes
+include::snippets/technology-preview.adoc[]
+
+You can view the procedures by selecting from one of the following providers:
+
+Amazon Web Services (AWS)
+
+* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.7/html/clusters/cluster_mce_overview#hosting-service-cluster-configure-aws[Configuring the hosted cluster on AWS]: Create the AWS S3 OIDC secret, create a routable public zone, enable external DNS, enable AWS PrivateLink, enable the hosted control planes feature, and install the hosted control planes CLI.
+* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.7/html/clusters/cluster_mce_overview#hosted-control-planes-manage-aws[Managing hosted control plane clusters on AWS]: Create, import, access, or delete a hosted cluster on AWS.
```

Comment:
The second two bullets feel more like complete sentences than the first two. I don't have a clever suggestion or anything but I wonder if they could all sound more similar :)

Reply:
Good point. I updated those two sentences a little to (hopefully) make them sound more like the others.